ext/protos_test.go

```diff
@@ -42,12 +42,21 @@ func TestProtos(t *testing.T) {
 		{expr: `proto.getExt(ExampleType{}, google.expr.proto2.test.nested_example) == ExampleType{}`},
 		{expr: `proto.getExt(ExampleType{}, google.expr.proto2.test.ExtendedExampleType.extended_examples) == []`},
 		{expr: `proto.getExt(ExampleType{}, google.expr.proto2.test.ExtendedExampleType.enum_ext) == GlobalEnum.GOO`},
+		{expr: "ExampleType{`in`: 64}.`in` == 64"},
+		// TODO(jnthntatum): legal parse, but can't assign extension fields without updates to type checker and runtime
```

Comment:
This is interesting. Are you thinking of doing this in a follow-up PR?

Reply:
Done. I added a reference, can you re=approve?